nvi

```diff
@@ -10,7 +10,12 @@ N=$(tput sgr0) # Normal
 U=$(tput smul) # Underline
 
 print_usage() {
-  cat << EOF
+  less \
+    --raw-control-chars \
+    --no-init \
+    --quit-if-one-screen \
+    --QUIT-AT-EOF \
```

Comment:
We can also simply remove `--QUIT-AT-EOF` and leave `less` open. That's just another way to violate conventions. :)